src/ext/handlersToResp.c

```diff
@@ -380,73 +382,131 @@ static RdbRes toRespFunction(RdbParser *p, void *userData, RdbBulk func) {
 
 /*** Handling raw ***/
 
+static RdbRes toRespRawBeginModuleAux(RdbParser *p, void *userData, RdbBulk name, int encver, int when, size_t size) {
+    char encstr[10];
+    UNUSED(p);
+
+    /* reset rawCtx */
+    RdbxToResp *ctx = userData;
+    ctx->rawCtx.rawSize = size;
+    ctx->rawCtx.sentFirstFrag = 0;
+    ctx->rawCtx.isModuleAux = 1;
+    ctx->rawCtx.crc = 0;
+
+    /* if target doesn't support module-aux, then skip it */
+    if (!ctx->conf.supportRestoreModuleAux)
+        return RDB_OK;
```

Comment:
It will add more complexity to responses pipeline tracking that I am not sure if it is worth it.
Another option is to extend boolean variable `supportRestore` to be:
 0=No support, 1=Support restore, 2=Support Restore with Module Aux

In the tradeoff of spamming the API with weird flags and deep hiding such feature, this sound 
most reasonable to me.

Reply:
i prefer not to add an interface at all (just silently ignore), but if we don't want that, then a separate interface would be easier to remove later (rather than mixing it with another interface).

i think at some point you'll have to have the complexity of understanding responses and associating them with the commands you sent anyway, and once you do that, then this optimistic approach won't add a big overhead.

one example is that when a command fails, i think we must log which command it was, not just the error message, otherwise, trouble shooting will be very difficult.

Reply:
Note that librdb can generate RESP output. How we suppose to decide whether to generate this special command or not? in that case, I prefer to leave the flag as is.

BTW, I do have some basic mechanism for pipeline to report of which command caused the error. Example to 100 pending list commands and failure on `mylist62` because the test makes "SET mylist62 1" before
```
:: ERROR :: [elementQuickList::State=1] 
Received Server error: "WRONGTYPE Operation against a key holding the wrong kind of value"
Got failed on command [#9501] (First 40 bytes):
*3
$5
RPUSH
$8
mylist62
$19
0.035

Preceding command [#9500] was: 
*3
$5
RPUSH
$8
mylist30
$17
0.791
```

Reply:
good point about sending RESP to a file rather than redis.
but then i wonder what do people do with that file? if they pipeline it to redis later (e.g. with `netcat`), then maybe they ignore replies (and errors) anyway.

regarding sending it to a live server, good that you have that feature in the error report.. but that means that it's not too complicated to have it implicitly ignore some errors (like unknown command, only on a certain command).

anyway, my point is that at some point in time we'll want to drop that interface, so i rather not have it if we can avoid it.